### DISCUSSION TITLE
Model has no relation to include

### DISCUSSION THREAD

**Discussion by jonathanwilke** [prisma/prisma#8312] [Q&A] [answered]
Hello everyone,

I'm trying to build a relation between to models with the following schema:

```graphql
model Assistant {
  id        Int             @id @default(autoincrement())
  title     String
  published Boolean         @default(false)
  createdAt DateTime        @default(now())
  uuid      String          @unique @default(uuid())
  steps     AssistantStep[]
}

model AssistantStep {
  id          Int                  @id @default(autoincrement())
  position    Int
  title       String?
  assistantId Int?
  createdAt   DateTime             @default(now())
  assistant   Assistant?           @relation(fields: [assistantId], references: [id])
  fields      AssistantStepField[]
}

model AssistantStepField {
  id        Int            @id @default(autoincrement())
  stepId    Int?
  createdAt DateTime       @default(now())
  step      AssistantStep? @relation(fields: [stepId], references: [id])
}
```

The code to load the assistant looks like this:

```typescript
await ctx.prisma.assistant.findUnique({
  where: {
    id,
  },
  include: {
    steps: true,
  },
});
```

This always results in the following error:
 
```
Invalid `prisma.assistant.findUnique()` invocation:
{
  where: {
    id: 2
  },
  include: {
    steps: true
  }

Unknown field `steps` for include statement on model Assistant.
This model has no relations, so you can't use include with it.
```

I'm on version 2.27.0 of Prisma.

**Comment by jonathanwilke** [comment, marked as answer]
The issue was caused by a cached version of the generated prisma client when running `vercel dev`. I had to restart the dev server to make it work.

**Reply by janpio** [reply]
Possibly a case of https://github.com/prisma/prisma/issues/7291

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
